crates/bevy_reflect/src/reflect.rs

```diff
@@ -137,11 +141,11 @@ pub trait Reflect: Any + Send + Sync {
 
     /// Clones the value as a `Reflect` trait object.
     ///
-    /// When deriving `Reflect` for a struct or struct tuple, the value is
-    /// cloned via [`Struct::clone_dynamic`] (resp.
-    /// [`TupleStruct::clone_dynamic`]). Implementors of other `Reflect`
-    /// subtraits (e.g. [`List`], [`Map`]) should use those subtraits'
-    /// respective `clone_dynamic` methods.
+    /// When deriving `Reflect` for a struct, tuple struct, or enum, the value is
+    /// cloned via [`Struct::clone_dynamic`], [`TupleStruct::clone_dynamic`],
+    /// or [`Enum::clone_dynamic`], respectively.
+    /// Implementors of other `Reflect` subtraits (e.g. [`List`], [`Map`]) should
+    /// use those subtraits' respective `clone_dynamic` methods.
```

Comment:
```suggestion
    /// use those subtrait's respective `clone_dynamic` methods.
```
I'm not too sure about this one, tho. I think it is a typo which was already present on previous code, so I assume original author intended `subtrait's`, but it could be `subtraits` also.

Reply:
I believe this is actually correct. The plural possessive form of `subtrait` would be `subtraits'`

Reply:
Agreed with @MrGVSV